tests/support/server.rs

```diff
@@ -23,6 +23,7 @@ pub struct Txn {
     pub read_timeout: Option<Duration>,
     pub response_timeout: Option<Duration>,
     pub write_timeout: Option<Duration>,
+    pub chunk_size: Option<usize>,
```

Comment:
You don't get **any** data before the response is complete. You get a chunk of the whole response after you send everything.